src/Data/Foreign/NullOrUndefined.purs

```diff
@@ -28,3 +28,5 @@ unNullOrUndefined (NullOrUndefined m) = m
 readNullOrUndefined :: forall a. (Foreign -> F a) -> Foreign -> F (NullOrUndefined a)
 readNullOrUndefined _ value | isNull value || isUndefined value = pure (NullOrUndefined Nothing)
 readNullOrUndefined f value = NullOrUndefined <<< Just <$> f value
+
+foreign import writeUndefined :: Foreign
```

Comment:
You can use `undefined` if you escape it as `exports["undefined"] = undefined;` on the JS side, I think.